reference/configuration/framework.rst

```diff
@@ -1372,6 +1372,14 @@ cache
 The service that is used to persist class metadata in a cache. The service
 has to implement the :class:`Symfony\\Component\\Validator\\Mapping\\Cache\\CacheInterface`.
 
+.. versionadded:: 2.8
+
```

Comment:
I'm currently doing a lot of work making the current docs compatible with the bot.

As for the bot, I'm stumbling on more and more edge cases. At the moment, the bot is just reviewing plain text files, I have the feeling that I have to make it somewhat context aware (having the bot as custom sphinx builder/visitor isn't possible).
I'm also planning on limiting reports for the diff of the PR only.